noetic/release-arm64-build.yaml

```diff
@@ -15,8 +15,17 @@ notifications:
   - sloretz+buildfarm@openrobotics.org
   maintainers: true
 package_blacklist:
-  - fetch_drivers
-  - ros_ign_gazebo
+- fetch_drivers
+- libntcan
+- mocap_nokov
+- nao_meshes
+- pepper_meshes
+- picovoice_driver
+- prosilica_gige_sdk
+- ros_ign_gazebo
+- ros_speech_recognition
+- rosrt
+- skyway
```

Comment:
One convention I really like from the ROS 2 build farm config is that build failures are annotated with a comment linking to the upstream issue reporting the failure: https://github.com/ros2/ros_buildfarm_config/blob/ros2/rolling/release-rhel-build.yaml#L19

If you have filed such issues it would be fantastic to adopt the same convention here.

Reply:
Added upstream issue links in 4c0aed8